src/components/manage/Tiles/Text/Edit.jsx

```diff
@@ -273,7 +273,7 @@ export default class Edit extends Component {
           this.state.customTilesOpened && (
             <div className="add-tile toolbar">
               {tiles.customTiles.map(tile => (
-                <Button.Group>
+                <Button.Group key={tile.title}>
```

Comment:
This is nice, but maybe it belongs to another PR, for separation of concerns.

Reply:
gotcha.